src/containers/Login.vue

```diff
@@ -38,75 +38,59 @@
-              <div class="col-1 tasks">
-                <img src="@/assets/images/Task4.svg" alt="Starred comment" />
-              </div>
-              <div class="col-11">
-                <p class="sub sub--task">Recommends courses based on your needs</p>
-              </div>
-            </div>
+          <div class="col tasks">
+            <figure class="figure">
+              <img src="@/assets/images/landing_new/icon1.svg" class="sub--task" alt="checklist" />
+              <figcaption class="sub">Fully personalized to track your requirements</figcaption>
+            </figure>
           </div>
-          <div class="col-6 col-md-6 image-wrapper women-wrapper">
-            <img
-              class="hide women"
-              src="@/assets/images/Person_planning.svg"
-              alt="women planning"
-            />
+          <div class="col tasks">
+            <figure class="figure">
+              <img src="@/assets/images/landing_new/icon2.svg" class="sub--task" alt="browser" />
+              <figcaption class="sub">Customizable interface to view your courses</figcaption>
+            </figure>
+          </div>
+          <div class="col tasks">
+            <figure class="figure">
+              <img src="@/assets/images/landing_new/icon3.svg" class="sub--task" alt="Network" />
+              <figcaption class="sub">Built-in system to check your progress</figcaption>
+            </figure>
+          </div>
+          <div class="col tasks">
+            <figure class="figure">
+              <img
+                src="@/assets/images/landing_new/icon4.svg"
+                class="sub--task"
+                alt="Starred comment"
+              />
+              <figcaption class="sub">View course information all in one place</figcaption>
+            </figure>
           </div>
         </div>
       </div>
 
       <!-- DRAG -->
-      <div class="container-fluid plan p-0">
+      <div class="container-fluid plan p-0 center">
         <div class="drag phonepad row no-gutters">
-          <div class="col-md-6 image-wrapper image-wrapper--drag">
-            <img
-              style="position: relative"
-              class="hide preview"
-              src="@/assets/images/drag.svg"
-              alt="Dragging preview"
-            />
-          </div>
-          <div class="col-12 col-md-6 comment">
-            <h1 class="head">Drag Your Course In</h1>
-            <p class="sub">
+          <img
+            class="hide preview"
+            src="@/assets/images/landing_new/img1.svg"
+            alt="Dragging preview"
+            style="position: absolute"
+          />
+          <div class="comment comment-drag">
+            <h1 class="head">Drag and Drop Courses</h1>
+            <p class="sub-text">
               CoursePlan’s intuitive interface recommends courses based on unfulfilled requirements
               and allows you to easily drag and drop them into your planner
```

Comment:
I think this should be aligned more to the right and also I get that this is as per Figma spec but it looks absolutely massive on my browser

Reply:
Can you post a screenshot?

Reply:
![image](https://user-images.githubusercontent.com/33106747/112768878-2920c780-8fec-11eb-9732-9a9a24bdaa13.png)

cursor in bottom right for comparison